packages/arb-token-bridge-ui/src/hooks/useTransactionHistory.ts

```diff
@@ -225,48 +248,64 @@ const useTransactionHistoryWithoutStatuses = (
     cctpTransfersTestnet.isLoadingDeposits ||
     cctpTransfersTestnet.isLoadingWithdrawals
 
+  const fetcher = useCallback(
+    (type: 'deposits' | 'withdrawals') => {
+      const fetcherFn = type === 'deposits' ? fetchDeposits : fetchWithdrawals
+
+      return Promise.all(
+        multiChainFetchList
+          .filter(chainPair => {
+            if (isTestnetMode) {
+              // in testnet mode we fetch all chain pairs
+              return chainPair
+            }
+            // otherwise don't fetch testnet chain pairs
+            return !isTestnetChainPair(chainPair)
+          })
+          .map(async chainPair => {
+            try {
+              return await fetcherFn({
+                sender: address,
+                receiver: address,
+                l1Provider: getProvider(chainPair.parentChain),
+                l2Provider: getProvider(chainPair.chain),
+                pageNumber: 0,
+                pageSize: 1000
+              })
+            } catch (err) {
+              const isCustomOrbitChain =
+                typeof getCustomChainFromLocalStorageById(chainPair.chain) !==
+                'undefined'
+
+              if (isCustomOrbitChain) {
```

Comment:
I think we should just throw and surface the error tbh, they can always remove the chain

Reply:
updated